inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -708,18 +732,21 @@ linelist_cleaned <- linelist_cleaned %>%
     exit_status == "Left against medical advice"              ~ "Left"
   ))
 
-## recode a character variable
-## fix any misspellings in the manually entered geographic region variables
-## you can choose any geographic region which you have available to you 
-## ideally this variable would fit your shapefile and population data! 
-## If you have many regions - consider group by an overarching unit (e.g. province)
+
+
+## Recode character variables -------------------------------------------------
+## This step shows how to fix misspellings in the geographic region variable.
+## Ideally, you want these values to match your shapefile and population data!
+
 # linelist_cleaned <- linelist_cleaned %>%
 #   mutate(patient_origin_free_text = fct_recode(patient_origin_free_text,
 #     # List all incorrect mis-spellings here ("new" = "old")
 #     "Village D" = "Valliages DD",
 #     "Village D" = "Villiage D"
 #   ))
 
+
+
 ## sometimes, coding is inconsistent across variables -- for example, "Yes" / "No"
```

Comment:
go in factor section? forcats::fct_recode...

Reply:
actually, patient_origin_free_text is not a factor -- should be a case_when, no?

Reply:
Good point... this forcats::fct_recode will generate a factor, which doesn't particularly make sense for a free text field

Reply:
actually this shouldnt even be using the free_text option, theres a variable i think just called patient_origin that isnt free text, use everywhere else

Reply:
still not a factor though -- will update varname